client/src/components/VersionInfo.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <p>
+      Version info:
+    </p>
+    <p v-for="component in versioninfo" :key="component.name">{{component.name}}
+        {{component.version}}</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+const API_URL = 'http://localhost:4000';
+
+export default defineComponent({
+  name: 'VersionInfo',
+  data() {
+    return {
+      versioninfo: [],
+    };
+  },
+  mounted() {
+    this.getVersions();
+  },
+  methods: {
+    async getVersions() {
+      try {
+        const response = await fetch(`${API_URL}/version`);
```

Comment:
Is this intention to move this fetch into Vuex in a separate ticket? Generally we would avoid components making API calls directly, allowing Vuex actions to handle this, and having the components invoke the actions and display Vuex state.